bindings/wasm/examples-account/src/create_vp.ts

```diff
@@ -101,16 +169,19 @@ async function createVP(clientConfig) {
     const resolver = await Resolver
         .builder()
         .clientConfig({
-            network: clientConfig.network
+            network: Network.mainnet()
         })
         .build();
 
     // Validate the presentation and all the credentials included in it according to the validation options
     await resolver.verifyPresentation(
         presentation,
         presentationValidationOptions,
-        FailFast.FirstError
+        FailFast.FirstError,
+        undefined,
+        undefined
```

Comment:
Why pass `undefined` here? Omitting the optional parameters is fine in Typescript.

Reply:
Should we fix it in this PR since it's a non-breaking change?

Reply:
Fixed by #807.